modules/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-dns-zone-infoblox_{context}"]
+= Creating DNS records on a public DNS zone on Infoblox
+
+You can create DNS records on a public DNS zone on Infoblox by using the Red Hat External DNS Operator.
+
+.Prerequisites
+
+* You have an access to OpenShift CLI (oc).
+* You have an access to Infoblox UI.
+
+.Procedure
+
+. Create a `secret` object with Infoblox credentials by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator create secret generic infoblox-credentials --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=<INFOBLOX_USERNAME> --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=<INFOBLOX_PASSWORD>
+----
+
+. Get the routes objects to check your cluster domain by running the following command:
++
+[source,terminal]
+----
+$ oc get routes --all-namespaces | grep console
+----
++
+.Example Output
+[source,terminal]
+----
+openshift-console          console             console-openshift-console.apps.test.example.com                       console             https   reencrypt/Redirect     None
+openshift-console          downloads           downloads-openshift-console.apps.test.example.com                     downloads           http    edge/Redirect          None
+----
+
+. Create an `ExternalDNS` resource YAML file, for example, sample-infoblox.yaml, as follows:
++
+[source,yaml]
+----
+apiVersion: externaldns.olm.openshift.io/v1beta1
+kind: ExternalDNS
+metadata:
+  name: sample-infoblox
+spec:
+  provider:
+    type: Infoblox
+    infoblox:
+      credentials:
+        name: infoblox-credentials
+      gridHost: ${INFOBLOX_GRID_PUBLIC_IP}
+      wapiPort: 443
+      wapiVersion: "2.3.1"
+  domains:
+  - filterType: Include
+    matchType: Exact
+    name: test.example.com
+  source:
+    type: OpenShiftRoute
+    openshiftRouteOptions:
+      routerName: default
+----
+
+. Create an `ExternalDNS` resource on Infoblox by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sample-infoblox.yaml
+----
+
+* From the Infoblox UI, check the DNS records created for `console` routes:
+
+** Click *Data Management* -> *DNS* -> *Zones*.
+** Select the zone name.
```

Comment:
With these now part of the normal flow, they should be numbered:

```suggestion
. From the Infoblox UI, check the DNS records created for `console` routes:
.. Click *Data Management* -> *DNS* -> *Zones*.
.. Select the zone name.
```

Reply:
Addressed